tests/integration_test/integration_test.sh

```diff
@@ -509,6 +509,16 @@ function stop_lira_on_error {
     docker rm -v ${LIRA_DOCKER_CONTAINER_NAME}
 
     print_style "error" "Test failed!"
+    print_style "error" "Please check if the following workflows failed:"
+    if [ -z "${SS2_WORKFLOW_ID}" ]; then
```

Comment:
I believe the `-z` option means that the variable is not defined...do you mean to use `-n` here? https://unix.stackexchange.com/questions/109625/shell-scripting-z-and-n-options-with-if

Reply:
Good catch! I also realized that I forgot `fi`s, a symptom of a Python developer writing bash 🤦‍♂️ Fixed now!